crates/viewer/re_viewer_context/src/tensor/image_stats.rs

```diff
@@ -187,3 +188,21 @@ impl ImageStats {
         }
     }
 }
+
+fn max_finite_range_for_datatype(datatype: ChannelDatatype) -> (f64, f64) {
```

Comment:
This could be a member function on `ChannelDatatype` so that it is easier to find and reuse

Reply:
:+1:

Reply:
turns out we already have min & max values there!